src/Marten/Events/EventDocumentStorage.cs

```diff
@@ -267,6 +256,43 @@ public async Task<IEvent> ResolveAsync(DbDataReader reader, CancellationToken to
 
         public abstract Task ApplyReaderDataToEventAsync(DbDataReader reader, IEvent e, CancellationToken token);
 
+        private EventMapping SetEventMappingForDotNetTypeName(string dotnetTypeName, string eventTypeName)
```

Comment:
It's nitpicking, but can you find a different name for this method that's more descriptive? This name implies a write, but it's not precisely that

Reply:
I dropped `Set` prefix from the method. I was struggling tbt how to name it, as it's not a pure method and also setting the Event Mapping internally if it doesn't exist.